b2luigi/core/utils.py

```diff
@@ -230,7 +230,8 @@ def create_output_file_name(task, base_filename, result_dir=None):
         result_dir = map_folder(get_setting("result_dir", task=task, default=".", deprecated_keys=["result_path"]))
 
     param_list = [f"{key}={value}" for key, value in serialized_parameters.items()]
-    output_path = os.path.join(result_dir, *param_list)
+    excaped_param_list = [param.replace("/", ".") for param in param_list]
```

Comment:
That `replace` could also be done in-place inside the first list comprehension in the line above, but for readability I thought it's better to split it up and have a separate statement on a separate line.

Reply:
There is a typo: "excaped" -> "escaped" (I guess this is what you wanted to say)

Reply:
Yes, meant that.